resources/views/account/edit.blade.php

```diff
@@ -96,3 +110,53 @@
         {{ Form::close() }}
     </div>
 @stop
+
+@push('scripts')
+<script type="text/javascript">
```

Comment:
This is complex enough that I think I'd like to pull it out to a separate JS file that goes through Elixir and then gets included here.